opacus/data_loader.py

```diff
@@ -28,6 +29,21 @@
 logger = logging.getLogger(__name__)
 
 
+def collate(
+    batch: List[torch.Tensor],
+    collate_fn: Optional[_collate_fn_t],
+    sample_empty_shapes: Sequence[Tuple],
+    dtypes: Sequence[Union[torch.dtype, Type]],
+):
```

Comment:
nit: Now that this is no longer an inner function, may be some docstrings?

Reply:
good point

Reply:
Thanks. Although, the style of docstrings is different from the ususal style.

Reply:
I relied on the IDE too much, thanks for catching!